schemesVoteParams4Production/FundingRequestParams.json.txt

```diff
@@ -0,0 +1,14 @@
+{ // settings for funding request
+  "boostedVotePeriodLimit": 518400, // is 6 days
+  "daoBountyConst": 1, // absolutely minimal dowstake (would be nice if we could turn this off)
+  "minimumDaoBounty": 1, // (the contract does not allow 0)
+  "queuedVotePeriodLimit": 100800, // the proposal expires in 4 weeks
+  "queuedVoteRequiredPercentage": 51,
+  "preBoostedVotePeriodLimit": 0, // no preboosted state at all
+  "proposingRepReward": 0, // no rep flow
+  "quietEndingPeriod": 86400, // 1 day
+  "thresholdConst": 1200, // ?? what is a good setting here?
+  "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+  "votersReputationLossRatio": 10,
+  "activationTime": 0
```

Comment:
this should be the fundingGoalDeadLine

Reply:
yes we need to change the script

Reply:
https://github.com/daostack/common-factory/issues/12